### DISCUSSION TITLE
iOS screencast – donations welcome to help prioritize this feature

### DISCUSSION THREAD

**Discussion by n00bsi** [rustdesk/rustdesk#4839] [Feature Request] [answered]
Hello,
will there a chance to Remote vom a PC/Latop to a iPhone / iOS Device ?

From iPhone/iOS to PC works but opposite not or did I not find the setting for ?

**Comment by Dirk23** [comment]
I am also absolutely in favor of such an extension to the app. Anydesk has this feature built in and it's very practical

**Reply by rustdesk** [reply]
AnyDesk can control ios? touch/keyboard work? As far as I know, iOS does not allow this.

**Reply by Dirk23** [reply]
Anydesk Can Not Control, but i Can at least See the Screen of the iOS/iPadOS Device

**Comment by rustdesk** [comment]
iOS is not controllable. Apple does not provide API. Viewer is in our plan, but not prioritized.

**Reply by rustdesk** [reply, marked as answer]
If you'd like to help prioritize this feature, please consider donating with below link and leaving a message.

https://ko-fi.com/rustdesk

I'll post all donations here.

**Reply by michaeljclarkk** [reply]
Hi Rustdesk, 

Thanks for the awesome product.

As a licensed pro server user with many clients/users, all served well by the custom clients for Android/Windows/Mac - we would like to see the viewer only app priority level raised if possible. 

Other pro users will agree too

**Comment by PC-Support-Services** [comment]
I already "donate" nearly $300 a year to you via my pro subscription.

Take that money and make iOS mobile device support a premium feature.

**Reply by michaeljclarkk** [reply]
I think it's reasonable to let the team respond to our requests in due time. As your "critique" has nothing actionable or worth replying to. 

They are aware of the increased need for this feature.

When choosing a rdp solution, view only for ios is an assumed feature.

**Reply by PC-Support-Services** [reply]
Free versions I don't expect to being able to view iOS devices but as a Pro licence user I'd expect it.

Having the ability limited to Pro licences would get more subscribers for them as well.

**Reply by michaeljclarkk** [reply]
Its more of a logistics issue. They can't build a custom client like the others are built. It needs to come from the app store.

The way forward is to have an IPA, with the same ui, that you have to point to your server, that's view only. 

Not as a pro feature, but to suppliment the ecosystem.  As the users that will benifit from it are the pro users.

**Reply by PC-Support-Services** [reply]
That requires end users to jailbreak their i-devices and 99.99% of people won't / can't do that.

RD will have to bite the bullet and get their apps back in to the Google Play and Apple app stores.

**Comment by Onizzaku** [comment]
Hello everyone.
Our company needed remote technical support access to hundreds of devices, and after a bad experience with competitors, we set our sights on RustDesk. What a surprise it was that RustDesk does not have the ability to view the screen of remote devices on iOS. Is there such a feature in the paid versions (we are considering the maximum purchase plan), and if not, is it expected in the near future?

**Reply by PC-Support-Services** [reply]
Nothing yet that I know of and there's nothing extra in the Pro version. Mention of iOS support questions here tend to get shut down quickly by the staff.

I've taken to using Airdroid Cast to support iOS devices - almost free / cheap subscription and works well.

**Reply by michaeljclarkk** [reply]
One solution that we have used is facetime with screenshare - because we have IOS, but still. Need a screenshare on IOS from rustdesk

**Comment by gbakeman** [comment]
I'm also waiting for iOS Screen Share capability before considering RustDesk - kind of disappointing to see a lack of urgency on this. Any chance we can at least turn this into an Issue to get some momentum?
